go/src/koding/workers/janitor/exempt.go

```diff
@@ -35,16 +34,7 @@ func NewChecker(name string, fn func(*models.User, *Warning) (bool, error)) *Exe
 
 // IsUserPaidFn checks if user is paid or not. All paid users are exempt.
 func IsUserPaidFn(user *models.User, _ *Warning) (bool, error) {
-	account, err := modelhelper.GetAccount(user.Name)
-	if err == mgo.ErrNotFound {
-		return false, nil
-	}
-
-	if err != nil {
-		return false, err
-	}
-
-	return paymentapi.New("").IsPaidAccount(account)
+	return false, nil
```

Comment:
Please add a `TODO` for this function. (it shouldn't have`Fn` suffix, but I see it would conflict with global IsUserPaid checker).